src/errors.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
Why do you need this?

Reply:
Our linter complains that each javascript file should have at most one class.

Since this file is dedicated only to custom error classes, it wouldn't make sense to enforce this rule here.

Reply:
Why not remove this on the eslint configuration, since it makes more sense to disable this rule altogether rather than make an exception for this file.

Reply:
Looking deeper into this issue, we found that the `wallet lib` also has this rule.
It seems like a good idea to keep it until we decide differently for these projects